src/App.js

```diff
@@ -6,13 +6,15 @@ import Login from './Components/Login';
 import SignUp from './Components/SignUp';
 import './App.css';
 import Home from './Components/Home';
+import Player from './Components/Player';
```

Comment:
If Player is commented out (and we don't need it any more), you can get rid of this import. :)